stories/Organism/Text/PostHeader/PostHeader.jsx

```diff
@@ -1,17 +1,15 @@
-import React from 'react';
-import './postheader.scss';
-import { Breadcrumbcomponent } from '../../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs';
-import { Post } from '../../../Molecules/Text/Post/Post';
+import React from "react";
+import "./postheader.scss";
+import { Breadcrumbcomponent } from "../../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs";
+import { Post } from "../../../Molecules/Text/Post/Post";
```

Comment:
@Ashish-srij Please lint the code as it should have the single quotes

Reply:
@surbhi90 Done